scripts/prepare_sector_network.py

```diff
@@ -2185,6 +2185,23 @@ def add_biomass(n, costs):
             carrier="solid biomass transport",
         )
 
+    elif options["biomass_spatial"]:
```

Comment:
What if biomass is not spatially resolved (just one EU node for the biomass). Should the transport costs be added? If not one could make the rule to build the biomass transport costs conditional again. If yes, the generator should be added here as well.

Reply:
I see, haven't thought about this option. Will make it conditional again